BookingsApi/BookingsApi/Controllers/WorkAllocationsController.cs

```diff
@@ -0,0 +1,137 @@
+using System;
```

Comment:
Is it maybe worth to have only one controller for the WorkHours module? We have already WorkHoursController . I appreciate that the name is not perfect but maybe We can use this controller instead of creating a new one? Or maybe move the actions in the old controller to the new one so we can have only one controller responsible for the module?

Reply:
I intentionally separated the controllers because they're growing because two or more groups of functionality are bringing in their dependencies which one or more group would not need. For example work hours is it's own piece in my opinion (focusing on hours and allocation controller is focused on allocation) and has no need for something like the `HearingAllocationService`

Reply:
ok fair enough